openedx/features/course_experience/views/course_outline.py

```diff
@@ -71,6 +72,12 @@ def render_to_fragment(self, request, course_id, user_is_enrolled=True, **kwargs
         if page_context:
             context['self_paced'] = page_context.get('pacing_type', 'instructor_paced') == 'self_paced'
 
+        # We use the edx-when models directly because there isn't yet an API call for finding
+        # out if edx-when is managing dates for a specific course.
+        # We're using this flag to prevent old self-paced dates from leaking out on courses not
+        # managed by edx-when.
+        context['not_in_edx_when'] = not ContentDate.objects.filter(course_id=course_overview.id).exists()
```

Comment:
is_enabled_for_course() should do that, yeah?

Reply:
Herp-a-derp, I completely missed that function.